test/Operators/remapping.jl

```diff
@@ -491,4 +556,164 @@ end
+            nq2 = 3
+            space2 = make_space(domain, nq2)
+
+            # SE1 -> SE2
+            @test Operators.x_overlap(space2, space1) ==
+                  Operators.y_overlap(space2, space1)
+            R = LinearRemap(space2, space1)
+
+            I_true = [1, 2, 4, 5, 2, 3, 5, 6, 4, 5, 7, 8, 5, 6, 8, 9] # row
+            J_true = [1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4] # col
+            @test (I_true, J_true) == sparsity_pattern(R)
+            @test size(R.map) == operator_size(R)
+
+            @test conservative(R)
+            @test consistent(R)
+
+            # SE2 -> SE1
+            R = LinearRemap(space1, space2)
+            @test size(R.map) == operator_size(R)
+
+            @test conservative(R)
+            @test consistent(R)
+        end
+
+        @testset "Multiple elements" begin
+            @testset "Aligned Elements" begin
+                nq1 = 2
+                space1 = make_space(domain, 2, 2, 2)
+                nq2 = 3
+                space2 = make_space(domain, 3, 2, 2)
+
+                # SE1 -> SE2
+                @test Operators.x_overlap(space2, space1) ==
+                      Operators.y_overlap(space2, space1)
+                R = LinearRemap(space2, space1)
+
+                I_true = [1, 2, 4, 5, 2, 3, 5, 6, 4, 5, 7, 8, 5, 6, 8, 9]
+                I_true = [
+                    I_true...,
+                    (nq2^2) .+ I_true...,
+                    (nq2^2 * 2) .+ I_true...,
+                    (nq2^2 * 3) .+ I_true...,
+                ]
+                J_true = [1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4]
+                J_true = [
+                    J_true...,
+                    (nq1^2) .+ J_true...,
+                    (nq1^2 * 2) .+ J_true...,
+                    (nq1^2 * 3) .+ J_true...,
+                ]
+                @test (I_true, J_true) == sparsity_pattern(R)
+                @test size(R.map) == operator_size(R)
+                @test conservative(R)
+                @test consistent(R)
+
+                # SE2 -> SE1
+                R = LinearRemap(space1, space2)
+                @test size(R.map) == operator_size(R)
+                @test conservative(R)
+                @test consistent(R)
+            end
+
+            @testset "Unaligned Elements" begin
+                # Square element layout
+                nq1 = 2
+                nq2 = 2
+                space1 = make_space(domain, nq1, 2, 2)
+                space2 = make_space(domain, nq2, 3, 3)
+
+                Xov = Operators.x_overlap(space2, space1)
+                Yov = Operators.y_overlap(space2, space1)
+                @test Xov == Yov
+                I_true = [1, 2, 3, 2, 3, 4, 3, 4, 5, 4, 5, 6]
+                J_true = [1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4]
+                @test sparsity_pattern(Xov) == (I_true, J_true)
+
+                # SE1 -> SE2
+                R = LinearRemap(space2, space1)
+                I_true = [1, 2, 3, 4, 5, 7, 13, 14, 17,
```

Comment:
The formatter does a bad job here and turns this into a new line for each entry in the vector. Is there a better way @charleskawczynski ?

Reply:
Sorry, just saw this. We could turn the formatter off. See, e.g., [here](https://github.com/CliMA/TurbulenceConvection.jl/blob/main/post_processing/compute_mse.jl#L22-L60)